src/main/java/at/hannibal2/skyhanni/config/features/chat/StashConfig.kt

```diff
@@ -23,17 +24,33 @@ class StashConfig {
     var notice: String = ""
 
     @Expose
-    @ConfigOption(name = "Hide Added Messages", desc = "Hide the messages when something is added to your stash.")
-    @ConfigEditorBoolean
-    var hideAddedMessages: Boolean = true
+    @ConfigOption(name = "Hide Dupe Warnings", desc = "")
+    @Accordion
+    var hideDuplicateWarning: HideDuplicateWarningConfig = HideDuplicateWarningConfig()
+
+    class HideDuplicateWarningConfig {
+        @Expose
+        @ConfigOption(
+            name = "Hide Dupe Warnings",
```

Comment:
having the category and the feature the same name might not be the most logical here. how about renaming the text to enabled as well?